sal/tests/dataclass/unit/test_calculated_dimension.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import pytest
+
+from sal.dataclass import CalculatedDimension
+
+
+@pytest.mark.parametrize('length, start, step, expected',
+                         [(5, 1, 2, np.array([1., 3., 5., 7., 9.])),
+                          (3, 10, 10, np.array([10., 20., 30.])),
+                          (4, 6, -2, np.array([6., 4., 2., 0.]))])
```

Comment:
**suggestion:** Just to test edge cases, you could add runs with lengths of 0 and 1.

Reply:
Good point.  As length of 0 is not allowed I've created a separate test for this.